local-proxy/src/localServer.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+// tslint:disable:no-console
+import express from 'express';
+import { setupProxy } from './index';
+import program from 'commander';
+const NAME = 'reshuffle-local-server';
+program.option('-p, --port', 'listening port (3000 by default)');
+program.description(`Run the ${NAME} outside of create-react-app server`);
+
+function main() {
+  program.parse(process.argv);
+  // create the express app to route local requests
+  const app = express();
```

Comment:
There should be a way for the user to configure his own routes and middleware for this to be useful.

Reply:
Scratch this comment